obspy/core/stream.py

```diff
@@ -829,6 +829,24 @@ def get_gaps(self, min_gap=None, max_gap=None):
             # skip if is equal to delta (1 / sampling rate)
             if same_sampling_rate and nsamples == 0:
                 continue
+            # check if gap is already covered in trace before:
+            covered = False
+            # you only need to check previous traces because the traces are sorted
+            for prev_i in range(_i):
+                prev_stats = self.traces[prev_i].stats
```

Comment:
i think something like this but be a bit prettier:

```python
for tr in self.traces[:_i]:
    ...
```

Usually we want to avoid iterating over lists using an index number unless it can't be helped